SampleUsages/Program.cs

```diff
@@ -42,30 +77,48 @@ public void RunScenario(string scenarioFilePath)
             Console.WriteLine("Start running scenarios.");
             var counter = 0;
             var now = DateTime.UtcNow;
+            var repo = new TestRepository();
+
+            var scenario = repo.AddTestScenario(new ServerlessResultManager.TestScenario
+            {
+                Name = string.Format("Scenario-{0}-{1}", scenarioFilePath, now.ToString("s")),
+                StartTimeUtc = now
+            });
 
             foreach (var testScenario in testScenarios)
             {
                 Console.WriteLine($"Start running scenario for function {testScenario.FunctionName} {++counter}/{testScenarios.Count}.");
                 var testFolder = string.Format(testScenario.FunctionName);
                 Directory.CreateDirectory(testFolder);
```

Comment:
Since you already doing this here then you won't need to create the directory again in the TestScenario.cs and AzurePerformanceResultsProvider.cs classes.